gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1304 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Authors: Tobias Brigl <github.com/sezanzeb>, Alex Salles <alex.salles@gmail.com>,
+# Alex Loosley <aloosley@alumni.brown.edu>, Data Reply Munich
+# Copyright (C) 2021 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""Ensemble Latent Dirichlet Allocation (eLDA), a method of training a topic model ensemble.
+
+Extracts reliable topics that are consistently learned across multiple LDA models, for higher reproducibility
+and less noise. eLDA has the added benefit that the user does not need to know the exact number of topics the topic
+model should extract ahead of time.
+
+For more details read `our thesis <https://www.sezanzeb.de/machine_learning/ensemble_LDA/>`_.
```

Comment:
Use a full, normal citation please. "Our" is too cryptic for open source user documentation – who are "we"?

Reply:
I'll remove that sentence since there is a link in the `Citation` section already